ivy/functional/frontends/mindspore/ops/function/nn_func.py

```diff
@@ -36,3 +36,23 @@ def selu(input_x):
 @to_ivy_arrays_and_back
 def softsign(x):
     return ivy.divide(x, ivy.add(1, ivy.abs(x)))
+
+
+@with_supported_dtypes({"2.0.0 and below": "float16"}, "mindspore")
+@to_ivy_arrays_and_back
+def gumbel_softmax(logits, tau=1, hard=False, eps=1e-10, dim=-1):
```

Comment:
I think this function signature is wrong https://www.mindspore.cn/docs/zh-CN/r2.1/api_python/ops/mindspore.ops.gumbel_softmax.html?highlight=gumbel_softmax#mindspore.ops.gumbel_softmax. Looks like we don't need `eps`

Reply:
@sushmanthreddy we still need to remove `eps` here right?